BasicRxJavaSample/app/build.gradle

```diff
@@ -2,11 +2,10 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion build_versions.compile_sdk
-    buildToolsVersion build_versions.build_tools
 
     defaultConfig {
         applicationId "com.example.android.persistence"
-        minSdkVersion build_versions.min_sdk
+        minSdkVersion 21
```

Comment:
Nit: In other samples, this variable is in the `versions.gradle` file

Reply:
Yes, this is WAI for this particular sample.